checker/src/main/java/org/checkerframework/checker/initialization/InitializationVisitor.java

```diff
@@ -381,16 +396,18 @@ protected void reportMethodInvocabilityError(
             return;
```

Comment:
Can you add a comment, before the `receiverType` local variable, what this check is for?

Reply:
Done in [aed8e24](https://github.com/eisop/checker-framework/pull/621/commits/aed8e241bca2625501d1315ebbbb82b32c816dd4)